packages/@webex/plugin-meetings/src/meeting/index.ts

```diff
@@ -3775,6 +3790,44 @@ export default class Meeting extends StatelessWebexPlugin {
     }
   }
 
+  /**
+   * Callback called when a relay event is received from meeting LLM Connection
+   * @param {any} e Event object coming from LLM Connection
+   * @private
+   * @returns {void}
+   */
+  private processRelayEvent = (e: any): void => {
+    switch (e.data.relayType) {
+      case REACTION_RELAY_TYPES.REACTION:
+        if (
+          // @ts-ignore - config coming from registerPlugin
+          (this.config.receiveReactions || options.receiveReactions) &&
+          this.isReactionsSupported()
+        ) {
```

Comment:
I guess these values can not change during the meeting. 

Please register the `processRelayEvent` listener only when it is needed

```ts
        if (
          // @ts-ignore - config coming from registerPlugin
          (this.config.receiveReactions || options.receiveReactions) &&
          this.isReactionsSupported()
        ) {
           this.webex.internal.llm.on('event:relay.event', this.processRelayEvent);
        }
```

Note: you do not need the same condition for the `off` because it is OK to remove unregistered listener

Reply:
Discussed 1:1
<img width="1017" alt="image" src="https://user-images.githubusercontent.com/14194394/214332060-84c4cac4-b937-461b-8852-7b65e318bfef.png">


TLDR; = leave as is.